src/test/java/org/mariadb/jdbc/failover/AuroraFailoverTest.java

```diff
@@ -119,21 +120,21 @@ public void pingReconnectAfterRestart() throws Throwable {
             int masterServerId = getServerId(connection);
             stopProxy(masterServerId);
 
-            long stoppedTime = System.currentTimeMillis();
             try {
                 st.execute("SELECT 1");
             } catch (SQLException e) {
                 //normal exception
             }
             restartProxy(masterServerId);
-            long restartTime = System.currentTimeMillis();
+            long restartTime = System.nanoTime();
```

Comment:
Even though we are not concerned about time changes in unit tests, nano time must be used since System.currentTimeMillis() may varry slightly from internal nano time usage.  This seems more accurate than just making the time checks have a larger verification window.

FWIW, these tests were not failing for me before the change.  I just know that from experience failing to change this may cause random unit test failures.